client/dashboard/sites/index.tsx

```diff
@@ -148,6 +148,9 @@ const DEFAULT_LAYOUTS = {
 		fields: [ 'subscribers_count', 'status', 'backups', 'protect' ],
 		titleField: 'name',
 		descriptionField: 'URL',
+		layout: {
```

Comment:
Regardless of whether we keep the flag or not, I think ultimately, we should move away from the "layout" config property and instead just have top level flags/configs.

Reply:
I'd prefer if we don't change how `layout` works for now. That's a major refactor that doesn't bring much value and complicates making syncs from upstream.

Reply:
How is this a major refactor? 

I think right now, the layout is already a superficial key, the "semantic" aspect of it has been removed essentially when we introduced title, description and media fields.

Reply:
We still have layout-bound properties in there, like `badgeFields` and `styles`. I am hearing you're proposing to remove them from there? This adds complexity for us (sync from upstream) and I don't see much value. In a world where we could ignore upstream, I'd be more open to do whatever we want.

Reply:
Yeah, I get your point, I'm not asking to change this now, but I'm thinking we should avoid adding new properties there

Reply:
Yep, the suggestion here was to add the new option outside of the layout instead of removing it.